opentelemetry-distro/setup.cfg

```diff
@@ -50,9 +50,9 @@ where = src
 
 [options.entry_points]
 opentelemetry_distro =
-    distro = opentelemetry.distro:OpenTelemetryDistro
+    otel-distro = opentelemetry.distro:OpenTelemetryDistro
```

Comment:
Wonder if we really need OpenTelemetry in the name if it is already in the path. `opentelemetry.distro.Distro` sounds nicer to me but may be that's just me.

Reply:
same thing about the plugin name. Why `opentelemetry_distro.otel-distro` instead of `opentelemetry_distro.distro`. I know neither is perfect but is there a reason to repeat otel?

Reply:
I see what you're saying, I actually wouldn't have changed except for what I learned in this Contrib PR to Enforce 1 Distro/Configurator: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/571

In that PR, I want to log a useful error message if the user has _multiple_ distros installed, but when it was just `distro` I didn't know where that distro was combing from. I thought have a message like `Multiple distros were found: (otel-distro,aws-distro,splunk-distro)` would be better because you know where they are coming from?

I guess the user knows what they installed though so we can keep just `distro`.

Reply:
Sorry if I misunderstand, but is "plugin name" the same as "entry_point"? Actually you gave me a good idea :]

I print out the attributes available on the entry_point and say this:

```
otel-distro = opentelemetry.distro:OpenTelemetryDistro
{'attrs': ('OpenTelemetryDistro',),
 'dist': opentelemetry-distro 0.23.dev0 (/usr/local/lib/python3.9/site-packages),
 'extras': (),
 'module_name': 'opentelemetry.distro',
 'name': 'otel-distro'}
```

So I can use the `module_name` for my purpose in that Contrib repo. Will change this back!

Reply:
Yeah but in that case we don't need to include `distro` in it probably. It could just be `otel`, `aws`, `lightstep` etc but doesn't really matter.

Reply:
Agreed, I'll just use `module_name` because it's easiest and we can always change it later (try to delete the "distro" substring) if we want.